lib/whitehall/document_filter/options.rb

```diff
@@ -76,6 +77,13 @@ def options_for_topics
         @options_for_topics ||= StructuredOptions.new(all_label: "All policy areas", grouped: Classification.grouped_by_type)
       end
 
+      def options_for_people
+        @options_for_people ||= StructuredOptions.new(
+          all_label: "All people",
+          ungrouped: Person.all.sort_by(&:name).map { |o| [o.name, o.slug] }
```

Comment:
You could `.order(:name).pluck(:name, :slug)` here for performance goodness.

Reply:
@gpeng I haven't been able to get this to work. `:name` seems to return the forename and surname of the person from Person, which is then used to populate the Atom feed filter url.